### PR TITLE
Docs: Add NFTScan to block explorers

### PR DESCRIPTION
**Pull request by sSelmann**
### Description of the Changes

This PR adds NFTScan to block explorers.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against master branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1056)

### CODE REVIEW

**Comment by stoobie** [comment]
@sSelmann 

The default branch for this repo is now `main` instead of `master`. [Please follow these steps to update your local repos to use main instead of master from now on.](https://github.com/starknet-io/starknet-docs/discussions/964#discussioncomment-7982322)

**Comment by sSelmann** [comment]
Hi @stoobie I did I guess

File: components/Starknet/modules/tools/pages/ref_block_explorers.adoc [hunk 1]
```diff
@@ -13,5 +13,6 @@ The following block explorers provide information on Starknet.
 |Starkscan | link:https://starkscan.co[https://starkscan.co^]
 |ViewBlock | link:https://viewblock.io/starknet[https://viewblock.io/starknet^]
 |Voyager | link:https://voyager.online[https://voyager.online^]
-|oklink  | link:https://www.oklink.com/starknet[https://www.oklink.com/starknet^]                
+|oklink  | link:https://www.oklink.com/starknet[https://www.oklink.com/starknet^]   
+|NFTScan | link:https://starknet.nftscan.com/[https://starknet.nftscan.com/^]
```
Comment (odednaor) [inline components/Starknet/modules/tools/pages/ref_block_explorers.adoc:17]:
```suggestion
|NFTScan (NFT explorer) | link:https://starknet.nftscan.com/[https://starknet.nftscan.com/^]
```

**Comment by odednaor** [comment]
Added minor suggestion, otherwise lgtm

**Pull request merged**